design/comment-scanning.md

```diff
@@ -0,0 +1,110 @@
+# Polymer Analyzer - Comment Directives & Comment Annotations
+> Author @FredKSchott
+> Status: Draft
+> Last Update: 2017-04-25
+
+## Objective
+- To create a paradigm for formatting document metadata and instructions for analysis.
+- Immediately driving this discussion is the need to support polymer-linter directives which can enable/disable lint rules within entire documents and/or single sections of documents.
+
+## Goals
+- Allow the analyzer to surface the information contained within polymer-lint directives so that the linter can properly report & ignore warnings.
+- Create a more general definition for a directive that is easy-to-use for code authors and fits into the data model of the analyzer.
+- Create a more general definition for metadata (ex: @polymerElement) that is attached to an already-existing feature.
```

Comment:
Fair enough, what I'm trying to communicate here is that one of my goals is to clearly differentiate directives from other analysis-related-info contained in comments. 

Do you think this shouldn't be a goal of the document? I'd remove most references to Comment Annotations in that case. Or, is there a better way to phrase this goal?